adoc/attributes.adoc

```diff
@@ -25,11 +25,11 @@
 :base_os_version: 15 SP1
 :cilium_version: 1.5.3
 :envoy_version:
-:crio_version: 1.16.0
-:dex_version: 2.16.0
-:etcd_version: 3.3.15
+:crio_version: 1.17.0
+:dex_version: 2.32.0
```

Comment:
on the thing we reverted, it was 2.23.0, is that a typo, or intentional?

Reply:
reverted